api4/channel_test.go

```diff
@@ -1716,29 +1723,29 @@ func TestGetChannelMember(t *testing.T) {
 	th := Setup(t).InitBasic()
 	defer th.TearDown()
 	Client := th.Client
```

Comment:
For the sake of our sanity, I'd remove these assignments or change the name of the helper variable to `c` or something like that :stuck_out_tongue_closed_eyes: 

I wasn't expecting to find so many `Client := th.Client` around the tests when I wrote the proposal...